design/welcome-email.feature

```diff
@@ -0,0 +1,93 @@
+Feature: Welcome new subscribers
+  - As a store owner
+  - I want to send welcome emails
+  - So that I will connect with my new subscribers immediately
+
+  Entities:
+    Guest - A person who has created an order or signed up for a newsletter. Guest has not created
+      unique customer account, but has a distinct email address in Magento store.
+    Customer - A person who has created an account in store.
+    Newsletter Subscriber - Person who exists in newsletter subscribers list in Magento. Newsletter
+      subscriber can have subscribed and unsubscribed status.
+    Need to Confirm - A setting in Magneto store that requires subscribers to confirm their
```

Comment:
Shouldn't an entity be an actor (for example a user) of the system? If it is then you can change this to Pending Subscriber (or similar).